layouts/dashboard/main.html

```diff
@@ -0,0 +1,35 @@
+
+<link rel="stylesheet" href="//interactives.data.spotlightpa.org/2020/coronavirus/coronavirus-dashboard/assets/styles.css">
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ==" crossorigin="">
+<link rel="stylesheet" href="//interactives.data.spotlightpa.org/2020/coronavirus/coronavirus-dashboard/assets/spotlight.css">
```

Comment:
These links don't use insecure HTTP, so may as well set the href to https://interactives.data.spotlightpa.org…